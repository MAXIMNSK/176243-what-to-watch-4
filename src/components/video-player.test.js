import React from "react";
import renderer from "react-test-renderer";

import VideoPlayer from "./video-player.jsx";

const mock = {
  src: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
  picture: `https://m.media-amazon.com/images/M/MV5BNWIwODRlZTUtY2U3ZS00Yzg1LWJhNzYtMmZiYmEyNmU1NjMzXkEyXkFqcGdeQXVyMTQxNzMzNDI@._V1_.jpg`,
  id: `f1`,
  playStatus: false,
};

describe(`Video player snapshot tests`, () => {
  it(`First test player`, () => {
    const player = renderer.create(
        <VideoPlayer
          src={mock.src}
          poster={mock.picture}
          playStatus={mock.playStatus}
        />
    ).toJSON();

    expect(player).toMatchSnapshot();
  });
});
