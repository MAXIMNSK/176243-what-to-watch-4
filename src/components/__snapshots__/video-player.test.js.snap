// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Video player snapshot tests First test player 1`] = `
<video
  poster="https://m.media-amazon.com/images/M/MV5BNWIwODRlZTUtY2U3ZS00Yzg1LWJhNzYtMmZiYmEyNmU1NjMzXkEyXkFqcGdeQXVyMTQxNzMzNDI@._V1_.jpg"
  src="https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4"
/>
`;
